layouts/index.html

```diff
@@ -59,38 +59,64 @@ <h3> {{ .title }} </h3>
     <!-- ========================== END OF GOALS ========================== -->
 
 
-
-
-    <!-- ================================== START OF KEYNOTE SPEAKERS ============================== -->
-    <!--
-    <section class="section speakers">
-        {{ $speakers := .Site.Data.incoming_talks}}
-        <h2> {{ $speakers.title}} </h2>
-        <div class="container speakers__container">
-            <div class="swiper speakerSlider">
-                <div class="swiper-wrapper">
-                    {{ range $speakers.incoming_talk }}
-                        <article class="speaker swiper-slide">
-                            <div class="iframe__container">
-                                <img src="{{ .imgPath }}" alt="">
-                            </div>
-                            <div class="speaker__info">
-                                <h4> {{ .title }} </h4>
-                                <div class="speaker__description">
-                                    <p> {{ .shortDescription }} </p>
-                                </div>
-                                <a href="{{ .readMoreURL }}" class="btn btn-primary">Learn more</a>
-                            </div>
-                        </article>
-                    {{ end }}
+    <!-- ================================== START OF FEATURED EVENT ============================== -->
+    {{ $events := sort .Site.Data.events.event "eventOrder" }}
```

Comment:
<img width="1462" alt="image" src="https://user-images.githubusercontent.com/7213793/211214892-726264de-7fcc-40eb-afce-2d680eb4434b.png">

Can events be moved above goals?

Reply:
Here is a preview of both the style, could you please take a look and let me know one looks better?

Event at the top of the goals: https://purna135.github.io/pymcon-test/
Event at the bottom of the goals: https://deploy-preview-60--pymcon.netlify.app/

Reply:
Top of goals is better